### ISSUE TITLE
License for go-play?

### ISSUE THREAD

**Issue by jaygarcia** [OtherCrashOverride/go-play#31]
Hi, 

What's the license for go-play? I think the dual-core implementation for Video & audio buffers is perfect for a game engine that utilizes these optimized methods. Is the license for go-play MIT? 

If we decide to write an engine on top of this, we'd make it MIT like we did for our EVADE Arduboy games:
https://github.com/ModusCreateOrg/evade2

Thanks :D

**Comment by rickyzhang82** [comment]
go-play builds on top of several existing emulators. I know NES emulator roots from nofrendo. Its license is GPLv2.

But I'm not sure the license of the modification from go-play and its porting work to ESP32.

**Comment by OtherCrashOverride** [comment]
All the individual emulators have a GPLv2 license.  Since libraries are compiled and linked into a single binary, this means GPLv2 also applies to all code modifications and additions.

TL;DR = License is GPLv2

**Comment by jaygarcia** [comment]
Thanks folks. =) This is very helpful.
